apps/website/types/charge.ts

```
@@ -8,6 +10,19 @@ export interface Charge {
   twitchStreamName: string
 }
 
+export interface ChargeInstance extends Charge {}
+
+export interface ChargeService {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)